porting/examples/freertos_nrf5_sdk/pca10040/blank/armgcc/nimble_FreeRTOS_gcc_nrf52.ld

```diff
@@ -0,0 +1,51 @@
+/* Linker script to configure memory regions. */
+
+SEARCH_DIR(.)
+GROUP(-lgcc -lc -lnosys)
+
+MEMORY
+{
+  FLASH (rx) : ORIGIN = 0x0, LENGTH = 0x80000
+  RAM (rwx) :  ORIGIN = 0x20000000, LENGTH = 0x10000
+}
```

Comment:
Move HW specific memmap into chip-specific `include "nrf528xx.ld"` with appropriate flash / ram sizes?

Reply:
this is chip specific - it's in `pca10040` folder which is used as build target for nRF52 DK only